examples/affinity/cookie/nginx/sticky-ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: nginx-test
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/affinity: "cookie"
```

Comment:
Yes, that's correct. If you supply only affinity=cookie, the default alg and cookie name will be the set [here](https://github.com/kubernetes/ingress/pull/258/files/a158e5fc5ab8118c7a51614a1dc43f97cf91b12e#diff-53c96b98408669762f605fb8532bd4fdR33)